docs/source/release_notes.rst

```diff
@@ -6,6 +6,7 @@ Release Notes
         * Added multiple oversampling and undersampling sampling methods as data splitters for imbalanced classification :pr:`1775`
         * Added params to balanced classification data splitters for visibility :pr:`1966`
         * Updated ``make_pipeline`` to not add ``Imputer`` if input data does not have numeric or categorical columns :pr:`1967`
+        * Updated ``class_imbalance_data_check`` to better handle multiclass imbalances :pr:`1986`
```

Comment:
Really nit-picking now LOL but can we update this to ``ClassImbalanceDataCheck``